source/adam.cpp

```diff
@@ -345,6 +346,27 @@ class sheet_t::implementation_t : boost::noncopyable {
     priority_t name_to_priority(name_t name) const;
     void flow(cell_bits_t& priority_accessed);
 
+    /// Returns the output cell with the given name.
+    /// \pre the output cell must exist.
```

Comment:
```suggestion
    /// Returns the output cell with the given name.
    ///
    /// \pre A cell with that name exists.
```

Blank line after summary.
A precondition is a predicate statement.  Write in English what you'd put in the `assert()` parens.

Reply:
Accepted.